language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -668,7 +667,7 @@ runWithSandbox Sandbox{port} maybeAuth tid ls = runLedgerService ls' timeout (co
             Nothing -> ls
             Just authSpec -> setToken (makeSignedJwt authSpec tid) ls
 
-makeSignedJwt :: AuthSpec -> TestId -> Jwt
+makeSignedJwt :: AuthSpec -> TestId -> String
 makeSignedJwt AuthSpec{sharedSecret} tid = do
```

Comment:
You now have two functions called `makeSignedJwt`, one here and one in the integration tests. The one here prepends `Bearer` whereas the other one doesn’t which is confusing. It would probably be nice to factor this out into a function in the ledger bindings. At the very least the two should be made consistent but it might also be a good idea to factor this out into a function in the ledger bindings:

```
data SandboxJwtPayload = SandboxJwtPayload
  { admin :: Bool
  , actAs :: [Text]
  -- There are more fields here, see https://github.com/digital-asset/daml/blob/f50424253fdab43920b35d24c37009d2edbec265/docs/source/tools/sandbox.rst but at least for this PR I would leave it at these.
  }
makeSandboxJwt :: SandboxJwtPayload -> Signer -> String
makeSandboxJwt = …
```